commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -155,7 +155,27 @@
 	MEASURES_RNR_WITH_DUPLICATED_MEASURE_GUID(71, "Two or more different measure section measure reference and results have "
 		+ "the same measure GUID. Please ensure that each measure section measure reference and results do not have "
 		+ "the same measure GUID."),
-	PERFORMANCE_RATE_MISSING(72, "The Performance Rate is missing");
+	PERFORMANCE_RATE_MISSING(72, "The Performance Rate is missing"),
+	IA_SECTION_IA_MEASURE_DUPLICATE_REPORTING_PARAM(73, "A Reporting Parameter Act has been found in both the "
+		+ "IA section and in IA section measures. The IA Section must have one Reporting Parameters Act "
```

Comment:
should this say IA section and IA measures

Reply:
Yes. Ultimately, the Reporting Parameter Act can be specified once within the IA Section or specified in each individual IA measure. However, the spec does not allow for the Reporting Parameter Act to be specified in both the IA section and within the IA measures of the given IA section.